contrib/spec/podman.spec.in

```diff
@@ -389,7 +405,12 @@ BUILDTAGS=$BUILDTAGS make binaries docs
 
 %if %{with varlink}
 #untar contents for python-podman
-pushd contrib/python/dist
+pushd contrib/python/podman/dist
+tar zxf %{name}*.tar.gz
+popd
+
+#untar contents for python-pypodman
+pushd contrib/python/pypodman/dist
 tar zxf %{name}*.tar.gz
```

Comment:
should be `tar zxf py%{name}-%{version}-linux-$ARCH.tar.gz`

Reply:
we also need a line in the %files section (order doesn't really matter) perhaps right below the %{_bindir}/%{name} entry like so: `%{_bindir}/py%{name}`